flows/field.go

```diff
@@ -137,7 +126,6 @@ func (v *FieldValue) ToXJSON(env utils.Environment) types.XText {
 }
 
 var _ types.XValue = (*FieldValue)(nil)
```

Comment:
`FieldValue` is still a complex object that needs to reduce itself to a primitive... I want to get rid of this... but I'm a bit stumped on what to do with locations and generally whether locations and URNs are types at an excellent level or just formatting behaviors of `XText`